rollup/ui5-resolve.js

```diff
@@ -0,0 +1,118 @@
+const path = require("path");
+const fs = require("fs");
+const stream = require("stream");
+const { promisify } = require("util");
+const pipe = promisify(stream.pipeline);
+
+const convertAMDtoES6 = require("@buxlabs/amd-to-es6");
+const amdextract = require("amdextract");
+
+const log = require("@ui5/logger").getLogger("rollup-plugin-ui5-resolve");
+const normalizer = require("@ui5/project").normalizer;
+const resourceFactory = require("@ui5/fs").resourceFactory;
+
+
+module.exports = (options) => {
+
+    log.verbose(JSON.stringify(options, undefined, 2));
+
+    const filter = (id) => !options.exclude.includes(id);
+
+    return {
+
+        name: "ui5-resolve",
+
+        buildStart: async (buildOptions) => {
+            const project = await normalizer.generateProjectTree({
+                cwd: options.projectPath
+            });
+            this.dependencies = resourceFactory.createCollectionsForTree(project, {}).dependencies;
+
+            const resources = await this.dependencies.byGlob("/resources/sap/ui/fl/**");
```

Comment:
here the assets are hard coded again - or did you want to use this configuration from assets?